AcsfToolsCommands.php

```diff
@@ -146,40 +150,6 @@ public function sitesInfo() {
    * @aliases sfml,acsf-tools-ml
    */
   public function ml($cmd, $command_args = '', $command_options = '', $options = ['profiles' => '', 'delay' => 0, 'total-time-limit' => 0, 'use-https' => 0]) {
-
-    // drush 9 limits the number of arguments a command can receive. To handle drush commands with dynamic arguments, we try to receive all arguments in a single variable $args & try to split it into individual arguments.
```

Comment:
Did you change that on purpose?

Reply:
Looks like no. @alex-moreno I am going to create a branch on upstream repository for you so that you can raise final pr and test from there

Reply:
this is probably coming from a patch in a project, but it has slipped here by mistake, yes

Reply:
just checked with JK, this was part of the drush 10 changes.